app/forms/mappings/Mappings.scala

```diff
@@ -60,6 +60,11 @@ trait Mappings extends Formatters with Constraints {
                               args: Seq[String] = Seq.empty)(implicit ev: Enumerable[A]): FieldMapping[A] =
     of(enumerableFormatter[A](requiredKey, invalidKey, args))
 
+  protected def enumerable2024[A](requiredKey: String = "error.required",
```

Comment:
Same as above

Reply:
added TODO comment to track future removal